### PR TITLE
[fix] add a special note for GSSoC'23 contributors

### LINKED ISSUE #590: [OTHER] Updating Contributing Guidelines
### What would you like to share or ask?

I'm adding a special note for GSSoC contributors, so they can follow the Contributing rules effectively. 

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by CBID2**
## Fixes Issue
Closes #590 

## Changes proposed

I added a special note for GSSoC'23 contributors in the Issues section. 
## Screenshots

proof for issue

## Note to reviewers
Hey, @rupali-codes! :) I decided to create this special note to make it easier for us to determine if contributors are participating in GSSoC'23. Let me know what you think. :)

### CODE REVIEW

**Comment by Oyebolade** [comment]
It's a good idea

**Comment by rupali-codes** [comment]
Great Idea Chirssy, just replace the image with this one :)
image

**Comment by rupali-codes** [comment]
Also, added _Closes #590_ in the description :)

**Comment by CBID2** [comment]
> Also, added _Closes #590_ in the description :)

Sure

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -75,12 +75,16 @@ The subcategory object should follow this pattern
...
 ## Issues
 * Select an issue template from [issues](https://github.com/rupali-codes/LinksHub/issues/new/choose) tab. Otherwise, choose **Other** if it doesn't match what you're looking for.
-* When creating an issue, make sure you fill up all the fields properly.
+* When creating an issue, make sure you fill up all the fields properly. 
 * Make sure that you are NOT raising a **duplicate issue**.
-* If you want to work on the issue, please click on the *I am willing to work on this issue* checkmark.
+* If you want to work on the issue, please click on the *I am willing to work on this issue* checkmark. 
+* **Note:** If you are apart of GSSoC'23, please check the *I am a GSSoC'23 contributor* check mark as shown in the image below:  
+![issue checkmark for gssoc'23](https://github.com/CBID2/LinksHub-my-version-/assets/105683440/c31b245d-420a-4513-bb58-d6be8b2fcdb1)
+ **Do not create the PR unitl you have confirmed with the maintainers that you are participating in this program**
```
Comment (rupali-codes) [inline CONTRIBUTING.md:83]:
```suggestion
 * **Do not create the PR unitl you have confirmed with the maintainers that you are participating in this program** 
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -75,12 +75,16 @@ The subcategory object should follow this pattern
...
 ## Issues
 * Select an issue template from [issues](https://github.com/rupali-codes/LinksHub/issues/new/choose) tab. Otherwise, choose **Other** if it doesn't match what you're looking for.
-* When creating an issue, make sure you fill up all the fields properly.
+* When creating an issue, make sure you fill up all the fields properly. 
 * Make sure that you are NOT raising a **duplicate issue**.
-* If you want to work on the issue, please click on the *I am willing to work on this issue* checkmark.
+* If you want to work on the issue, please click on the *I am willing to work on this issue* checkmark. 
+* **Note:** If you are apart of GSSoC'23, please check the *I am a GSSoC'23 contributor* check mark as shown in the image below:  
+![issue checkmark for gssoc'23](https://github.com/CBID2/LinksHub-my-version-/assets/105683440/c31b245d-420a-4513-bb58-d6be8b2fcdb1)
+ * **Do not create the PR unitl you have confirmed with the maintainers that you are participating in this program**
```
Comment (rupali-codes) [inline CONTRIBUTING.md:83]:
```suggestion
 * **Do not create the PR until you have confirmed with the maintainers that you are participating in this program** 
```

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**